assets/helpers/page/page.js

```diff
@@ -32,8 +32,9 @@ import {
   detect as detectCountryGroup,
   type CountryGroupId,
 } from 'helpers/internationalisation/countryGroup';
+import { type OptimizeExperiments, getOptimizeExperiments } from 'helpers/tracking/optimize';
 
-import type { Action } from './pageActions';
+import { type Action } from './pageActions';
```

Comment:
Other files use the original pattern - how do you decide which one to use?

Reply:
AFAIK the original pattern only works for types, so if you're also importing values (e.g. functions) from a file, you have to add a separate `import` line for them. The new pattern allows you to import types and values in one line, so is potentially better for most use cases. @rupertbates discovered the new pattern a couple of months back so the original pattern still exists in many places.